app/src/main/java/com/vci/vectorcamapp/intake/presentation/IntakeScreen.kt

```diff
@@ -496,45 +487,29 @@ fun IntakeScreen(
                         })
 
                     if (surveillanceForm.wasIrsConducted) {
```

Comment:
I think this if condition is not required anymore if you are using 

surveillanceForm.monthsSinceIrs?.let